src/Compilers/VisualBasic/Portable/Binding/Binder_Conversions.vb

```diff
@@ -1042,6 +1042,15 @@ DoneWithDiagnostics:
 
             Dim tupleElements As BoundConvertedTupleElements = CreateConversionForTupleElements(tree, sourceType, targetType, convKind, isExplicit)
 
+            Dim targetNamedType = TryCast(targetType, NamedTypeSymbol)
+            If argument.IsNothingLiteral() AndAlso targetNamedType IsNot Nothing AndAlso targetType.IsStructureType() Then
+                ' Check to see if we're creating a default value of a structure with required members
```

Comment:
>' Check to see if we're creating a default value of a structure with required members [](http://example.com/codeflow?start=16&length=85)

I am not sure why are we checking this. It looks like C# doesn't care for similar scenario:
```
class C1
{
    S1 Test()
    {
        return default;
    }
}

struct S1
{
    required public int F;
}
```

Reply:
I was responding to your comment on a previous iteration, under the assumption that we would change the C# code. However, since we've since decided to spec C# as not requiring members for structs on `default`, I'll be reverting this part of the change and updating the tests.